bundles/org.openhab.core/src/test/java/org/openhab/core/internal/i18n/TestKelvinProvider.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2010-2025 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.core.internal.i18n;
```

Comment:
In my view, this has nothing to do with `i8n`. I understand you put the class there because there already is `TestUnitProvider`.  But I would keep it with the test you are doing.

Reply:
Done